lisa/transformers/dom0_kernel_installer.py

```diff
@@ -258,3 +262,25 @@ def _update_mariner_config(
         file=mariner_config,
         sudo=True,
     )
+
+    uuid = node.execute(
+        "lsblk -o MOUNTPOINT,UUID | grep '/ ' | awk '{print $2}'",
+        sudo=True,
+        shell=True,
+        expected_exit_code=0,
+    ).stdout.strip()
+    partuuid = node.execute(
+        "lsblk -o MOUNTPOINT,PARTUUID | grep '/ ' | awk '{print $2}'",
+        sudo=True,
+        shell=True,
+        expected_exit_code=0,
+    ).stdout.strip()
+
+    # initramfs can only understand PARTUUID
+    sed.substitute(
```

Comment:
This also will run for 2.0 as well. Do we need that?

Reply:
This is good to have for 2.0 as well, since PARTUUID is used in 2.0's config.